scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -60,11 +66,28 @@ class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig)
                   str("#")
               }
             }
+            if (config.better && pre == s.NoType) {
+              name map lookup match {
```

Comment:
One of the conventions that Olaf and I follow in Scalameta (I carried that to Rsc as well) is to avoid infix method applications unless the operator is symbolic. I suggest we do that in RscCompat too.